acp/source_hub_client.go

```diff
@@ -335,6 +356,83 @@ func (a *sourceHubBridge) CheckDocAccess(
 	}
 }
 
+func (a *sourceHubBridge) AddDocActorRelationship(
+	ctx context.Context,
+	policyID string,
+	resourceName string,
+	docID string,
+	relation string,
+	requestActor identity.Identity,
+	targetActor string,
+) (bool, error) {
+	// Must have all required args.
+	if policyID == "" ||
+		resourceName == "" ||
+		docID == "" ||
+		relation == "" ||
+		requestActor == (identity.Identity{}) ||
+		targetActor == "" {
+		return false, NewErrMissingRequiredArgToAddDocActorRelationship(
+			policyID,
+			resourceName,
+			docID,
+			relation,
+			requestActor.DID,
+			targetActor,
+		)
+	}
+
+	exists, err := a.client.AddActorRelationship(
+		ctx,
+		policyID,
+		resourceName,
+		docID,
+		relation,
+		requestActor,
+		targetActor,
+		protoTypes.TimestampNow(),
+	)
+
+	if err != nil {
+		return false, NewErrFailedToAddDocActorRelationshipWithACP(
+			err,
+			"Local",
+			policyID,
+			resourceName,
+			docID,
+			relation,
+			requestActor.DID,
+			targetActor,
+		)
+	}
+
+	if exists {
```

Comment:
todo: Duplicating the log between the if-else here really reduces the information density, and increases the amount of stuff maintainers have to read. Please simplify this to something similar to the below:
```
log.InfoContext(
			ctx,
			"Document and actor relationship set",
			corelog.Any("PolicyID", policyID),
			corelog.Any("ResourceName", resourceName),
			corelog.Any("DocID", docID),
			corelog.Any("Relation", relation),
			corelog.Any("RequestActor", requestActor.DID),
			corelog.Any("TargetActor", targetActor),
			corelog.Any("Existed", exists),
		)
		return exists, nil
```

Reply:
Good suggestion, will do.

- [ ] logs